.github/pull_request_template.md

```diff
@@ -1,17 +1,8 @@
-*Short paragraph summarizing the changes*
-
 ---
+title: ""
```

Comment:
markdownlint complains without this bit

Reply:
do you have the `line-length` option disabled? because it complains for 2 lines for me.

Reply:
Locally in your VSCode? No complaints for me with this change, I wonder if we have different global configs

Reply:
yeah, I just have the stock one and just using https://github.com/fannheyward/coc-markdownlint and by default the config is `{}` which has the https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md line length rule on (80 columns).

I'll disable mine locally.